docs/_newsfragments/2381.newandimproved.rst

```diff
@@ -0,0 +1,8 @@
+Added support for private network requests in :class:`falcon.middleware.CORSMiddleware`.
```

Comment:
Could you shorten the lines in this file?
We don't have a hard requirement for Rst, but it is nice if you could keep them around 80 chars too.